admin_manual/configuration_files/previews_configuration.rst

```diff
@@ -106,3 +106,16 @@ Default JPEG quality setting for preview images is '90'. Change this with:
 :: 
 
   occ config:app:set preview jpeg_quality --value="60"
+
+Maximum memory for image generation:
+^^^^^^^^^^^^^^^^^^^^^
+
+By default, image previews are generated with imagegd.
```

Comment:
```suggestion
```

This sounds confusing ;)

Reply:
I've added a commit to make the sentence more clear.